src/createServer.js

```diff
@@ -13,7 +12,6 @@ function createServer() {
   clearUsers();
   clearExpenses();
 
-  app.use(cors());
   app.use('/expenses', express.json(), expenseRouter);
   app.use('/users', express.json(), userRouter);
 
```

Comment:
just as a suggestion,
// To ensure proper JSON parsing, use express.json() middleware one time
// and define routes for users and expenses
app.use(express.json());
app.use('/users', userRouter);
app.use('/expenses', expenseRouter);